content/docs/command-reference/exp/show.md

```diff
@@ -21,34 +21,38 @@ usage: dvc exp show [-h] [-q | -v] [-a] [-T] [-A] [-n <num>]
 
 Displays experiments and
 [checkpoints](/doc/command-reference/exp/run#checkpoints) in a detailed table
-which includes their parent and name (or hash), as well as project metrics and
-parameters. Only the experiments derived from the Git `HEAD` are shown by
-default but all experiments can be included with the `--all-commits` option.
-Example:
+which includes their parent and name (or hash), as well as colored columns for
+(left to right): metrics (yellow), parameters (blue) and
+<abbr>dependencies</abbr> (violet).
+
+Only the experiments derived from the Git `HEAD` are shown by default but all
+experiments can be included with the `--all-commits` option. Example:
 
 ```dvc
 $ dvc exp show
 ```
 
 ```dvctable
- ───────────────────────────────────────────────────────────────────
-  neutral:**Experiment**      metric:**avg_prec**   metric:**roc_auc**   param:**train.n_est**  param:**train.min_split**
- ───────────────────────────────────────────────────────────────────
-  workspace        0.56191   0.93345   50           2
-  master           0.55259   0.91536   50           2
-  ├── exp-bfe64    0.57833   0.95555   50           8
-  └── exp-ad5b1    0.56191   0.93345   50           2
- ───────────────────────────────────────────────────────────────────
+ ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  neutral:**Experiment**                  neutral:**Created**        metric:**avg_prec**   metric:**roc_auc**   param:**featurize.max_features**   dep:**model.pkl**   dep:**data/features**
+ ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+  workspace                   -               0.60405    0.9608   3000                     484fab5     52c1fdd
+  random-forest-experiments   May 29, 2021    0.60405    0.9608   3000                     484fab5     52c1fdd
+  ├── a2efdc9 [exp-68ac9]     10:21 PM        0.55669   0.93516   1000                     e2b5a9a     1b2d542
+  ├── e7bd029 [exp-25e9a]     10:21 PM        0.58589     0.945   2000                     7aae464     2ac217b
+  └── 56f3be3 [exp-8f5c4]     10:21 PM        0.51799   0.92333   500                      cfbfed4     64ed644
+ ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 ```
 
 Your terminal will enter a
 [paginated screen](/doc/command-reference/dag#paginating-the-output) by default,
 which you can typically exit by typing `Q`. Use `--no-pager` to print the table
 to standard output.
 
-By default, the printed experiments table will include columns for all metrics
-and params from the entire project. The `--only-changed`, `--drop`, `--keep`,
-and other [options](#options) can determine which ones should be displayed.
+By default, the printed experiments table will include columns for all
+dependencies, metrics and params from the entire project. The `--only-changed`,
```

Comment:
In general, should we order in the same way they appear (metrics, params, deps) throughout the docs?

Reply:
I changed the order. 

I have also noticed that `params` and `parameters` are used arbitrarily. 

Should we consolidate `params` / `deps` or the longer version? cc @jorgeorpinel

Reply:
I usually prefer the longer versions thinking they are more likely to be searched.

Reply:
Replaced all occurrences with long version